packages/flutter/lib/src/material/stepper.dart

```diff
@@ -722,8 +776,10 @@ class _StepperState extends State<Stepper> with TickerProviderStateMixin {
           Expanded(
             child: Container(
               margin: const EdgeInsets.symmetric(horizontal: 8.0),
-              height: 1.0,
-              color: Colors.grey.shade400,
+              height: height,
+              color: (widget.steps[i + 1].isActive)
+                    ? activeColor
+                    : inactiveColor,
```

Comment:
Instead of specifying it like this, you could use a `connectorColor` that was a  `MaterialStateProperty<Color?>`, and would resolve it for each step.

```suggestion
              color: widget.connectorColor.resolve(<MaterialState>[
                if (!widget.steps[i + 1].isActive) MaterialState.disabled,
              ]),
```

Reply:
Thanks for the feedback, I have made the changes you requested.